.github/workflows/build-and-deploy.yml

```diff
@@ -0,0 +1,34 @@
+name: Tag repo; build and publish assets
+on:
+  create:
```

Comment:
Also want `workspace_dispatch:` here, as I guarantee you it won't work first time! :)